homeassistant/components/roku/remote.py

```diff
@@ -60,9 +66,13 @@ async def async_turn_off(self, **kwargs: Any) -> None:
     async def async_send_command(self, command: Iterable[str], **kwargs: Any) -> None:
         """Send a command to one device."""
         num_repeats = kwargs[ATTR_NUM_REPEATS]
+        delay = kwargs.get(ATTR_DELAY_SECS, DEFAULT_DELAY_SECS)
```

Comment:
Is this a breaking change?  The behavior before was no delay now it is a 0.4 second delay.  Should we default to zero instead?

Reply:
I'd second this. I see no reason it should default to something other than 0.

While from my experience the API doesn't guarantee the order of commands and adding a delay can help, adding a delay still isn't a guarantee of order and even more so since it would make this a breaking change.

Reply:
With the Broadcom integration (aka an IR blaster), it has a similar method of specifying a delay as the target device may not respond to very rapid IR commands.  As they are designed to respond to someone hitting the volume up button - so maybe 5x a second.  So you'd set the delay to 200ms and repeat to 5.